src/Subscriptions.tsx

```diff
@@ -29,6 +29,7 @@ export interface SubscriptionProps<TData = any, TVariables = OperationVariables>
   shouldResubscribe?: any;
   client?: ApolloClient<Object>;
   onSubscriptionData?: (options: OnSubscriptionDataOptions<TData>) => any;
+  onSubscriptionComplete?: () => any;
```

Comment:
Since we're not doing anything with the return value of `onSubscriptionComplete`, let's change the `any` return type to `void`.